src/models/ChatMessage.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+
+const chatMessageSchema = new mongoose.Schema(
+  {
+    albumId: {
+      type: String,
+      required: true,
+    },
+    message: {
+      type: String,
+      required: true,
+    },
+    userId: {
```

Comment:
Let's call this field `user` . Usually we call `ref` fields by the model we're referring to, rather than the `id` even though we can save it with just the id. See for example the `ref` properties that we have on the `Order` and `OrderItem` models: [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/ef73cbab610ba7d19ae351e6366edd3082a0ecb2/src/models/Order.js#L11](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/ef73cbab610ba7d19ae351e6366edd3082a0ecb2/src/models/Order.js#L11)

```suggestion
    user: {
```

It also makes it a bit clearer then that we can use the [populate](https://mongoosejs.com/docs/populate.html) method in the album chat socket handler to get the user name so that we can return that data in the socket.

```javascript
// in `handleAlbumChat.js`

		const msg = await ChatMessage.create({ ...data, user: data.userId });
		await msg.populate('user', 'name'); // this populates the "user" property and because of the `ref` property on the model, mongoose knows  that it should lookup this id in the User schema. The second argument 'name', tells it to return _only_  that field.
```